mitiq/lre/multivariate_scaling/layerwise_folding.py

```diff
@@ -208,3 +209,8 @@ def multivariate_layer_scaling(
         multiple_folded_circuits.append(folded_circuit)
 
     return multiple_folded_circuits
+
+
+multivariate_layer_scaling = accept_qprogram_and_validate(
+    _multivariate_layer_scaling, one_to_many=True
+)
```

Comment:
@natestemen Can we spend next week's coding session discussing these circuit converter decorators? When I was struggling with this, I knew I had to use `one_to_many=True`. But I struggled to use it with ` accept_qprogram_and_validate`.